clang/test/SemaTemplate/concepts.cpp

```diff
@@ -1151,3 +1151,17 @@ int test() {
 }
 
 }
+
+namespace GH109780 {
+
+template <typename T>
+concept Concept; // expected-error {{expected '='}}
+
+bool val = Concept<int>; // expected-error {{use of undeclared identifier 'Concept'}}
```

Comment:
It could be improved as to the confusing language 'undeclared identifier'.

We probably could continue as if the concept was defined as `= false`, but I'm not sure there's much value in it.